test/bin/test-citgm-all.js

```diff
@@ -17,17 +17,6 @@ test('citgm-all: /w markdown /w -j', function (t) {
   });
 });
 
-test('citgm-all: envVar /w -J', function (t) {
```

Comment:
this env var test spawns a shell script and checks if an environment variable has been set